addons/l10n_id/models/res_bank.py

```diff
@@ -67,3 +89,18 @@ def _get_qr_code_generation_params(self, qr_method, amount, currency, debtor_par
                 'value': self._get_qr_vals(qr_method, amount, currency, debtor_partner, free_communication, structured_communication),
             }
         return super()._get_qr_code_generation_params(qr_method, amount, currency, debtor_partner, free_communication, structured_communication)
+
+    def make_qris_request(self, url, params):
```

Comment:
```suggestion
    def _make_qris_request(self, url, params):
```
No reason for this to be public